### ISSUE TITLE
Anatomy and Physiology - bullets in lists in tables falling out of the cell

### ISSUE THREAD

**Issue by yblock** [openstax/cnx-recipes#4528]
# Brief Description of Scenario and Screenshots
bulleted list inside of table item renders with bullet outside of the cell in PDF

<img width="653" alt="Screen Shot 2022-03-08 at 12 08 08 PM" src="https://user-images.githubusercontent.com/5865292/157299169-78e3bdcd-96e0-4a77-924d-a15ce1ae747a.png">

# Does this happen for every instance? Is there a discernable pattern of occurrences? Please describe below.
Appears to happen in all cases of bulleted lists within table cells

# Acceptance Criteria
bullet point placement will be adjusted so that it falls within its respective cell. 

# Sample mark up if applicable
```
 <list id="fs-id1491159" list-type="bulleted"><item>Secretes several hormones, including hCG, estrogens, and progesterone, to maintain the pregnancy and stimulate maternal and fetal development</item>
```

# Location(s)
github openstax/osbooks- link: https://github.com/openstax/osbooks-anatomy-physiology
collection slug (if applicable): anatomy-and-physiology
chapter/section: 
surrounding text: Secretes several hormones, including hCG

**Comment by KarinaMendez2** [comment]
There's a very global style being applied to `[data-type="page"] ul`'s that sets `padding: 0;`, this is affecting all scenarios in the book, by removing this padding, the screenshoted sections in this card get fixed but other areas like: 

![image.png](https://images.zenhubusercontent.com/5c54789f0a972e6fd1ece1a0/21e8555a-d18a-475a-af40-31e1ce859971)

get affected. I also found a comment for this book saying:

![image.png](https://images.zenhubusercontent.com/5c54789f0a972e6fd1ece1a0/68453357-7776-4a41-8e5d-3b5674869bb8)

I suggest a styles cleanup for lists instead of targeting one by one scenarios.

**Comment by ajluk** [comment]
@lwtchu I have added styles for lists in tables based on suggestion in a comment in code that the margin should be smaller in tables lists than in the lists in the rest of the book. There has been a function created for margins in tables - `table-h-spacing`. When I set it to `1` the margin is equal to 0.7rem (8.39px) but it doesn't look good: 
![image](https://user-images.githubusercontent.com/55203598/160160198-9d00f92b-6bd6-460a-a462-432f421ec935.png)
so I changed it to table-h-spacing(2) which is equal to 1.4rem (16.79px). Here is how it looks: 
![image](https://user-images.githubusercontent.com/55203598/160160523-6e9e8d75-59e1-4b6d-a7a5-b7df138a2dc9.png)
Is it ok?
